### PR TITLE
Fix Room Card styling

### PR DESCRIPTION
**Pull request by scouillard**
## Description

Add border-radius to the square icon
Change button size from normal to large
Change copy button color from muted to brand color

## Screenshots (if appropriate):

From
image

To
image

### CODE REVIEW

**Comment by Davka** [comment]
why don't you use the variable border-radius?

**Comment by scouillard** [comment]
> why don't you use the variable border-radius?

Thank you for the heads up.

File: app/assets/stylesheets/variables.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 $secondary: #4B4B4B;
-$border-radius: 8px;
+$border-radius: 10px;
```
[the remark this answers is not in the document]
Comment (KH-Amir-TN) [reply app/assets/stylesheets/variables.scss:2]: If it's global then I guess that we need to also update the rest of bootrap's border-radius variables accordingly, 
[More](https://getbootstrap.com/docs/5.0/utilities/borders/#variables)

File: app/assets/stylesheets/rooms.scss [hunk 2]
```diff
@@ -1,5 +1,6 @@
 #room-card {
   .room-icon-square {
```
Comment (KH-Amir-TN) [inline app/assets/stylesheets/rooms.scss:2]: I'd change the class name to `room-icon` instead of `room-icon-square`,

**Review by KH-Amir-TN — DISMISSED** [review]

File: app/javascript/components/rooms/RoomCard.jsx [hunk 3]
```diff
@@ -20,7 +20,7 @@ export default function RoomCard({ room }) {
   return (
     <Card id="room-card" className="h-100 shadow-sm border-0">
       <Card.Body className="pb-0" onClick={handleClick}>
-        <div className="room-icon-square rounded-3">
+        <div className="room-icon-square">
```
[the remark this answers is not in the document]
Comment (scouillard) [reply app/javascript/components/rooms/RoomCard.jsx:23]: It does not work for some reason.

[the remark this answers is not in the document]
Comment (KH-Amir-TN) [reply app/javascript/components/rooms/RoomCard.jsx:23]: I think it worked for me, will check in the meeting

File: app/assets/stylesheets/variables.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (scouillard) [reply app/assets/stylesheets/variables.scss:2]: It's definitely global as every corners in the mock-ups is the same.

[the remark this answers is not in the document]
Comment (KH-Amir-TN) [reply app/assets/stylesheets/variables.scss]: Then we should further update the rest of bootstrap `border-radius` variables respecting their ratios,

[the remark this answers is not in the document]
Comment (scouillard) [reply app/assets/stylesheets/variables.scss]: It's been done

**Review by farhatahmad — APPROVED** [review]

**Pull request merged**